src/audio/drc/drc_math_lib/db2mag/matlab_fixpt/drc_db2mag_fixpt.m

```diff
@@ -0,0 +1,27 @@
+function y = drc_db2mag_fixpt(tstruct)
```

Comment:
Is this Matlab coder source file? A new directory e.g. tools/reference/<your work>/ might be suitable.